Source/String+Kakapo.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  String+Kakapo.swift
+//  KakapoExample
+//
+//  Created by Joan Romano on 31/03/16.
+//  Copyright © 2016 devlucky. All rights reserved.
+//
+
+import Foundation
+
+func randomString(withLength len: Int = 20) -> String {
+    let allowedChars = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
```

Comment:
Might be used when creating serializable structs, as the examples in the unit tests.